src/main/resources/assets/js/app/wizard/page/LiveFormPanel.ts

```diff
@@ -745,6 +748,45 @@ export class LiveFormPanel
         });
     }
 
+    private canMarkContentAsReady(componentPath: ComponentPath): boolean {
+        const isReady = this.content.isReady();
+
```

Comment:
Let's exit the method if `isReady` is false, instead of wrapping the entire body inside `if...`

```
private canMarkContentAsReady(componentPath: ComponentPath): boolean {
  if (!this.content.isReady()) {
    return false;
  }
  const persistedContent = this.contentWizardPanel.getPersistedItem();
     ...
  return viewedPage.equals(serverPage);
}
```